meetings/2024-01-10.md

```diff
@@ -75,33 +90,65 @@ URL: https://virginiabalseiro.github.io/solid-ecosystem-monitor/
-
+* FG: FG: Enabling information interoperability by thinking of a pod as the complete space-time model for a part of the world where the pod has two interconnecting hierarchies: one hierarchy of files/documents in ldp containers with root at `/`; and a second hierarchy of system relations with root at the webid for the system that the pod represents.  A demo of this approach is in action using a Car Dealership building on a TrinPod.
+* MdJ: Am working on documenting using your ontology. Regarding private attributes and search, question for the group: we're sort of re-organised now, who would pick up on this - can we evolve solid to those things that are needed in practise?
+* eP: Great presentation. Nice UI. Important for UX, and nice demos to see in Solid. Seems you have specific knowledge domain, for a lot of people also doing clone of X clone, and Solid perhaps more complex, or the scope the broader than just social media. Great to see use cases like yours. Regarding sub-resources access control, would like to hear about your experiences on that. What kind of intearctions work for the user and how the document hierarchy interacts with the knowledge hierarchy?
+* FG: We have CSV upload. Simplified way to put triples into the system. A lot of problems to solve for this to work. One is ontologies. This creates extra bandwidth but we deliver an onotology where each time you make a request, it also delivers the (complete) ontology needed. We only create what doesn't exist. Our system ontology worked for that. We have conceptual "ids", things like "t_type" are just there as shortcut. This would be for a sophisticated user to set this up. You can of course do all this with RDF, this was a quick and easy UI. Also have commands in a row-based instructions in a spreadsheet. We'll increasingly add automation. The ontology that would allow interoperability.
+* SC: Do you have a link to the ontology? In one sentence how are you separating public from private statements in the same resource and how you manage that?
+* FG: We have Linked Data Hypergraph. What we've done is another layer on top of RDF to effectively have states, where it has a unique URI. Connects an entity with attribute to a thing that's true for a certain time. So we put an ACL on that. https://porsche.trinpod.us/i is a `vcard:WebId`. We look at the resource, and see many (e.g., 32) states for the entity. Has a public ACL on them. Everything is like this in the pod. Default is not public. Only when needed for Solid interoperability. For Group pods, it owuld be based on the intent of the user. The idea would to having a URI representing a triple. However you do it, it'd be helpful. The problem with LD is we don't know context for that triple, who created, when.. or why it is true, and have history of events showing what caused it. It just supports human-info.
```

Comment:
```suggestion
* SC: Do you have a link to the ontology? In one sentence, how are you separating public from private statements in the same resource, and how do you manage that?
* FG: We have Linked Data Hypergraph. What we've done is another layer on top of RDF to effectively have states, where it has a unique URI. Connects an entity with attribute to a thing that's true for a certain time. So we put an ACL on that. <https://porsche.trinpod.us/i> is a `vcard:WebId`. We look at the resource, and see many (e.g., 32) states for the entity. Has a public ACL on them. Everything is like this in the pod. Default is not public. Only when needed for Solid interoperability. For Group pods, it would be based on the intent of the user. The idea would be to have a URI representing a triple. However you do it, it'd be helpful. The problem with LD is we don't know context for that triple (e.g., who created it, when or why it is true) and have history of events showing what caused it. It just supports human-info.
```